packages/icons/build/index.js

```diff
@@ -0,0 +1,90 @@
+module.exports = /******/ (function(modules) {
+  // webpackBootstrap
+  /******/ // The module cache
+  /******/ var installedModules = {} // The require function
+  /******/
+  /******/ /******/ function __webpack_require__(moduleId) {
+    /******/
+    /******/ // Check if module is in cache
+    /******/ if (installedModules[moduleId]) {
+      /******/ return installedModules[moduleId].exports
+      /******/
+    } // Create a new module (and put it into the cache)
+    /******/ /******/ var module = (installedModules[moduleId] = {
+      /******/ i: moduleId,
+      /******/ l: false,
+      /******/ exports: {},
+      /******/
+    }) // Execute the module function
+    /******/
+    /******/ /******/ modules[moduleId].call(
+      module.exports,
+      module,
+      module.exports,
+      __webpack_require__
+    ) // Flag the module as loaded
+    /******/
+    /******/ /******/ module.l = true // Return the exports of the module
+    /******/
+    /******/ /******/ return module.exports
+    /******/
+  } // expose the modules object (__webpack_modules__)
+  /******/
+  /******/
+  /******/ /******/ __webpack_require__.m = modules // expose the module cache
+  /******/
+  /******/ /******/ __webpack_require__.c = installedModules // identity function for calling harmony imports with the correct context
+  /******/
+  /******/ /******/ __webpack_require__.i = function(value) {
+    return value
+  } // define getter function for harmony exports
+  /******/
+  /******/ /******/ __webpack_require__.d = function(exports, name, getter) {
+    /******/ if (!__webpack_require__.o(exports, name)) {
+      /******/ Object.defineProperty(exports, name, {
+        /******/ configurable: false,
+        /******/ enumerable: true,
+        /******/ get: getter,
+        /******/
+      })
+      /******/
+    }
+    /******/
+  } // getDefaultExport function for compatibility with non-harmony modules
+  /******/
+  /******/ /******/ __webpack_require__.n = function(module) {
+    /******/ var getter =
+      module && module.__esModule
+        ? /******/ function getDefault() {
+            return module['default']
+          }
+        : /******/ function getModuleExports() {
+            return module
+          }
+    /******/ __webpack_require__.d(getter, 'a', getter)
+    /******/ return getter
+    /******/
+  } // Object.prototype.hasOwnProperty.call
+  /******/
+  /******/ /******/ __webpack_require__.o = function(object, property) {
+    return Object.prototype.hasOwnProperty.call(object, property)
+  } // __webpack_public_path__
+  /******/
+  /******/ /******/ __webpack_require__.p = '' // Load entry module and return exports
+  /******/
+  /******/ /******/ return __webpack_require__((__webpack_require__.s = 0))
+  /******/
+})(
+  /************************************************************************/
+  /******/ [
+    /* 0 */
+    /***/ function(module, exports) {
+      throw new Error(
```

Comment:
is this working ?

Reply:
Probably not. Do you remember we decided to divide this PR into 2? This is the first one, which deals only with the "styleguide" matter. I should've removed this webpack.config.js file anyway and that build directory. After we finish this "styleguide" part, I'll take care of all the React/Vue package component creation stuff.